### PR TITLE
FALCON-1846 Fixing ProcessInstanceStatusTest,ProcessInstanceResumeTest,EntityDryRunTest

### PR DESCRIPTION
**Pull request by pragya-mittal**
FALCON-1846 Fixing ProcessInstanceStatusTest,ProcessInstanceResumeTest,EntityDryRunTest

### CODE REVIEW

File: falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java [hunk 1]
```diff
@@ -516,7 +516,7 @@ public static String getOutputFolderForInstanceForReplication(OozieClient oozieC
     private static String getReplicatedFolderFromInstanceRunConf(String runConf) {
         String inputPathExample = getReplicationFolderFromInstanceRunConf(runConf).get(0);
         String postFix = inputPathExample.substring(inputPathExample.length() - 7, inputPathExample.length());
-        return getReplicatedFolderBaseFromInstanceRunConf(runConf) + postFix;
+        return getReplicatedFolderBaseFromInstanceRunConf(runConf) + "/" + postFix;
     }
```
Comment (muraliramasami) [inline falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java]: @pragya-mittal  Can we use the OSUtil.concat() here?

Comment (pragya-mittal) [inline falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java]: Cannot use this function. It replaces all the occurrences of '//' by '/'. My input contains hdfs://randomPath which is converted to hdfs:/randomPath which is not the expected value.

Comment (pragya-mittal) [inline falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java]: Removing ProcessInstanceStatusTest changes as discussed.

Comment (muraliramasami) [inline falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java]: Thanks @pragya-mittal

**Comment by muraliramasami** [comment]
Thanks @pragya-mittal  . +1 Patch Looks good to me.

**Pull request closed without merging**